### ISSUE TITLE
`--ampls-resource-id` is not available in `aks-preview`

### ISSUE THREAD

**Issue by JoeyC-Dev** [Azure/azure-cli#29296]
### Describe the bug

`--ampls-resource-id` is not available in `aks-preview` only.

### Related command

`az aks enable-addons -a monitoring --ampls-resource-id`

### Errors

unrecognized arguments: --ampls-resource-id

### Issue script & Debug output

```
az extension add -n aks-preview
az aks enable-addons -a monitoring -n ${aks} -g ${rG} --workspace-resource-id ${logAnalyticsWorkspace_resId} --ampls-resource-id ${ampls} --debug`
```

```
...
cli.azure.cli.core.azclierror: unrecognized arguments: --ampls-resource-id 
az_command_data_logger: unrecognized arguments: --ampls-resource-id 
```

### Expected behavior

This argument is available if not using `aks-preview`. Should add to `aks-preview`.

### Environment Summary

```
{
  "azure-cli": "2.61.0",
  "azure-cli-core": "2.61.0",
  "azure-cli-telemetry": "1.1.0",
  "extensions": {
    "ai-examples": "0.2.5",
    "aks-preview": "5.0.0b3",
    "ml": "2.26.1",
    "ssh": "2.0.3"
  }
}
```

**Comment by yonzhan** [comment]
Thank you for opening this issue, we will look into it.

**Comment by orenzp** [comment]
I got the same issue,

I run the following command and was able to use `az aks enable-addons` with `--ampls-resource-id`

`az extension update --name aks-preview`

**Comment by FumingZhang** [comment]
The option was added to aks-preview since `7.0.0b3` by PR [#7754](https://github.com/Azure/azure-cli-extensions/pull/7754), and I would recommend you use version `7.0.0b4+` as there was a fix for this option by PR [#7863](https://github.com/Azure/azure-cli-extensions/pull/7863).

Please upgrade the aks-preview you've installed.
```
  "extensions": {
    "ai-examples": "0.2.5",
    "aks-preview": "5.0.0b3",
    ...
```
